Install Issue with Apple M2 Max chip

**Describe the bug**
I am a new employee at JCSDA and while running `spack install` on my laptop I received a ESMA_cmake error. My MacBook pro has the Apple M2 Max chip and running Ventura 13.0. I was using the skylab-dev template. Here is the error: 

```
1 error found in build log:
     83      ESMA_cmake/compiler/flags/GNU_Fortran.cmake:78 (ecbuild_warn)
     84      ESMA_cmake/compiler/esma_compiler.cmake:9 (include)
     85      ESMA_cmake/esma.cmake:52 (include)
     86      CMakeLists.txt:29 (include)
     87    
     88    
     >> 89    CMake Error at ESMA_cmake/compiler/flags/GNU_Fortran.cmake:140 (message):
     90      Unknown processor.  Please file an issue at
     91      https://github.com/GEOS-ESM/ESMA_cmake
     92    Call Stack (most recent call first):
     93      ESMA_cmake/compiler/esma_compiler.cmake:9 (include)
     94      ESMA_cmake/esma.cmake:52 (include)
     95      CMakeLists.txt:29 (include)
```

**To Reproduce**
I followed instructions at [spack-stack readthedocs](https://spack-stack.readthedocs.io/en/latest/Platforms.html#creating-a-new-environment)

**Expected behavior**
Clean install. 

**System:**
Run on Macbook Pro with Apple M2 Max chip and Ventura 13.0. I had my terminal in a i386 architecture.

--- a/.gitmodules
+++ b/.gitmodules
@@ -1,11 +1,9 @@
 [submodule "spack"]
   path = spack
-  ##url = https://github.com/spack/spack
-  ##branch = develop
-  #url = https://github.com/NOAA-EMC/spack
-  #branch = jcsda_emc_spack_stack
-  url = https://github.com/climbfuji/spack
-  branch = feature/update_esma_cmake_in_mapl
+  #url = https://github.com/spack/spack
+  #branch = develop
+  url = https://github.com/NOAA-EMC/spack
+  branch = jcsda_emc_spack_stack
 [submodule "doc/CMakeModules"]
   path = doc/CMakeModules
   url = https://github.com/noaa-emc/cmakemodules

--- a/.gitmodules
+++ b/.gitmodules
@@ -1,9 +1,11 @@
 [submodule "spack"]
   path = spack
-  #url = https://github.com/spack/spack
-  #branch = develop
-  url = https://github.com/NOAA-EMC/spack
-  branch = jcsda_emc_spack_stack
+  ##url = https://github.com/spack/spack
+  ##branch = develop
+  #url = https://github.com/NOAA-EMC/spack
+  #branch = jcsda_emc_spack_stack
+  url = https://github.com/climbfuji/spack
+  branch = feature/update_esma_cmake_in_mapl
 [submodule "doc/CMakeModules"]
   path = doc/CMakeModules
   url = https://github.com/noaa-emc/cmakemodules

